platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/TaskContainerExtensions.kt

```diff
@@ -276,5 +276,58 @@ inline fun <reified T : Task> TaskContainer.register(name: String, vararg argume
  * Creates a [Task] with the given [name] and type, passing the given arguments to the [javax.inject.Inject]-annotated constructor,
  * and adds it to this project tasks container.
```

Comment:
💅   The `@deprecated` doclet tag allows `@link`ing to the preferred API, which I think `@Deprecated` does not.

Reply:
But this is Kotlin not Java. And given the features of `@Deprecated` and my (limited) testing, there is no doc level tag. All the info comes from the annotation.

Reply:
Confirmed, no `@deprecated` in KDoc. See [here](https://kotlinlang.org/docs/kotlin-doc.html#block-tags).